webapp/src/components/Help.tsx

```diff
@@ -14,7 +14,7 @@ import { alpha, useTheme } from '@mui/material'
 const Help = () => {
```

Comment:
Import Container in line 12, and add:
```suggestion
import Stack from '@mui/material/Stack';

const Help = () => {
```

Reply:
Updated imports to include Container & Stack